plugins/outputs/exec/exec.go

```diff
@@ -26,13 +26,15 @@ const maxStderrBytes = 512
 
 // Exec defines the exec output plugin.
 type Exec struct {
-	Command     []string        `toml:"command"`
-	Environment []string        `toml:"environment"`
-	Timeout     config.Duration `toml:"timeout"`
-	Log         telegraf.Logger `toml:"-"`
+	Command        []string        `toml:"command"`
+	Environment    []string        `toml:"environment"`
+	Timeout        config.Duration `toml:"timeout"`
+	UseBatchFormat bool            `toml:"use_batch_format"`
+	Log            telegraf.Logger `toml:"-"`
 
 	runner     Runner
 	serializer serializers.Serializer
+	outBuffer  bytes.Buffer
```

Comment:
I'm not sure about this bit. This is how I'm getting at the output from the command that's been run in the unit tests, but it has no purpose outside of tests. Any better ideas?

Reply:
How about replacing the `runner` in your test and simply count the number of calls in that "mocked runner"?

Reply:
Hmm. This could probably work, I'll need to load this back up and see what I can do.

Reply:
@gazpachoking any update?

Reply:
Sorry, just been busy and haven't gotten back to this yet.

Reply:
Okay, switched to a mocked runner that just counts metrics and runs. I think this is a cleaner way. 👍